wlib/scite.h

```diff
@@ -1148,20 +1148,11 @@ class CSciTeExtension
         return true;
     }
 
-    void FoldAll() 
+    void FoldAll(bool expanding = false)
```

Comment:
`expanding` is not used - how can this work?

Reply:
It is used. I added "Fold All" and "Unfold All". See line 646 of ECLDlgBuilder.cpp: https://github.com/hpcc-systems/eclide/blob/b85053688fb11e94001891f0fa058d585ad1dc37/eclide/EclDlgBuilder.cpp#L646

Reply:
So it is - I didn't look far enough down...